node-fs-readfile/cat.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+
+const argv = process.argv;
+
+const [, , ...txtFiles] = argv;
+
+for (let i = 0; i < txtFiles.length; i++) {
+  fs.readFile(txtFiles[i], 'utf-8', (err, data) => {
+    if (err) {
+      console.log(err);
+    }
+    console.log(data);
+  });
+}
+
+console.log('txtFiles:', txtFiles);
```

Comment:
While this will read all of the files, the output won't be guaranteed. They will not always print out in the same order as what the user input them. Good attempt but not the full solution.